tasks/hivemind/query_data_sources.py

```
@@ -1,17 +1,15 @@
 import nest_asyncio
 from dotenv import load_dotenv
-from typing import Type, Optional
+from typing import Optional, Callable
 from tc_temporal_backend.client import TemporalClient
 from tc_temporal_backend.schema.hivemind import HivemindQueryPayload
-from pydantic import BaseModel, Field
 
 nest_asyncio.apply()
 
-from crewai.tools import BaseTool
+from langchain.tools import tool
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)